packages/runtime-common/indexer.ts

```diff
@@ -503,6 +529,126 @@ export class Indexer {
     ];
   }
 
+  async searchPrerendered(
+    realmURL: URL,
+    { filter, sort, page }: Query,
+    loader: Loader,
+    opts: QueryOptions = {},
+  ): Promise<{
+    prerenderedCards: PrerenderedCard[];
+    prerenderedCardCssItems: PrerenderedCardCssItem[];
+    meta: QueryResultsMeta;
+  }> {
+    let { results, meta } = await this._search(
+      realmURL,
+      { filter, sort, page },
+      loader,
+      opts,
+      'SELECT url, ANY_VALUE(file_alias) as file_alias, ANY_VALUE(embedded_html) as embedded_html',
+    );
+
+    let cardFileAliases = results.map((c) => c.file_alias);
+    let realmVersion = meta.page.realmVersion;
+
+    let conditions: CardExpression[] = [
+      ['i.realm_version = ', param(realmVersion)],
+      ['i.type =', param('instance')],
+      [
+        `i.file_alias IN (${cardFileAliases
```

Comment:
The cards returned by the 1st query are used to filter the 2nd query where we fetch css for them. Note to self: check if `param(cardFileAliases)` can be used here

Reply:
for using a `param()` you'd want to do it insdie the map